### DISCUSSION TITLE
Convert an Expression to a Dataframe

### DISCUSSION THREAD

**Discussion by argenisleon** [vaexio/vaex#2002] [Q&A]
Hi,

Is there a way to convert an Expression to a Vaex Dataframe(Virtual Column), something like `Pandas.Series.to_frame`?

**Comment by maartenbreddels** [comment]
Hi,

I'm not sure what your purpose is, but you can do:
```
df['my_column'] = ....
df_single_column = df[['my_column']]
```
Does that help?

Regards,

Maarten
